### PR TITLE
Allow render component inside backdrop

### PR DESCRIPTION
**Pull request by r0b0t3d**
## Motivation

This PR adds an option to render child component inside default backdrop

### CODE REVIEW

File: src/components/bottomSheetBackdrop/BottomSheetBackdrop.tsx [hunk 1]
```diff
@@ -104,7 +105,7 @@ const BottomSheetBackdropComponent = ({
     [disappearsOnIndex]
   );
   //#endregion
-
+
```
Comment (gorhom) [inline src/components/bottomSheetBackdrop/BottomSheetBackdrop.tsx]: could you remove this space ?

File: src/components/bottomSheetBackdrop/types.d.ts [hunk 2]
```diff
@@ -39,4 +39,8 @@ export interface BottomSheetDefaultBackdropProps
    * @default 'close'
    */
   pressBehavior?: BackdropPressBehavior;
+  /**
+   * Child component that will be rendered on backdrop.
+   */
+  children?: React.ReactNode;
```
Comment (gorhom) [inline src/components/bottomSheetBackdrop/types.d.ts]: could you add `React.ReactNode[]`

**Review by gorhom — CHANGES_REQUESTED** [review]
thanks @r0b0t3d for submitting this pr, i have left some comments , once they've been addressed i will merge it :)

File: src/components/bottomSheetBackdrop/types.d.ts (hunk 2 above)
Comment (r0b0t3d) [reply src/components/bottomSheetBackdrop/types.d.ts]: @gorhom you mean like this?
```
children?: React.ReactNode | React.ReactNode[];
```

Comment (gorhom) [reply src/components/bottomSheetBackdrop/types.d.ts]: yea

**Comment by r0b0t3d** [comment]
@gorhom I resolved comments, please help take a look. Thanks

**Review by gorhom — APPROVED** [review]

**Comment by gorhom** [comment]
thanks @r0b0t3d for updating the PR 👍

**Pull request merged**